packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -49,6 +49,28 @@ import 'typography.dart';
 //
 // 5. If you are a Google employee, you should then also follow the instructions
 //    at go/flutter-l10n. If you're not, don't worry about it.
+//
+// UPDATING AN EXISTING STRING
+//
+// If you (someone contributing to the Flutter framework) want to modify an
+// existing string in the MaterialLocalizations objects, follow these steps:
+//
+// 1. Modify the default value of the relevant getter(s) in
+//    DefaultMaterialLocalizations below.
+//
+// 2. Update the flutter_localizations package. Modify the out-of-date English
+//    strings in lib/src/l10n/material_en.arb.
+//
+//    You also need to re-generate lib/src/l10n/localizations.dart by running:
+//    ```
+//    dart dev/tools/localization/bin/gen_localizations.dart --overwrite
```

Comment:
When I ran this script in my other PR, it actually created a few labels in new English locales (like en ca) with the old values. I manually updated them to the correct values.  I guess this shouldn't be part of the process, though, and any new labels created in new locales with the old values should be translated like normal.

Reply:
Manually updating them actually caused an analyzer error, which gives me more confidence that you shouldn't do that :) Those strings will get translated and set to the right values later.

Reply:
maybe add a statement along these lines to the docs so that people don't get confused?

Reply:
Good call 👍